domains/aeyika.json

```diff
@@ -0,0 +1,14 @@
+{
+  "description"{
+    "description": "Domain for my portfolio website"
+  }
+  
+  "owner": {
+    "username": "aeyika",
+    "email": "aeyikastech@gmail.com",
+    "discord": "aeyika"
+  },
+  "record": {
+    "TXT": "_vercel.aeyika.is-a.dev"
```

Comment:
This TXT should be a domain name file, and a verification TXT value should be placed.

Refer to the docs: https://is-a.dev/docs/vercel/